Makefile

```diff
@@ -0,0 +1,15 @@
+help:
+	@echo "Usage:"
+	@echo "    make help        show this message"
+	@echo "    make setup       create virtual environment and install dependencies"
+	@echo "    make activate    enter virtual environment"
+	@echo "    exit             leave virtual environment"
+
+setup:
+	pip install pipenv
+	pipenv install --dev --three
```

Comment:
`--three` is not needed if you already have a Pipfile which specifies the requirement.